opm/simulators/wells/GasLiftRuntime_impl.hpp

```diff
@@ -354,8 +372,14 @@ runOptimizeLoop_(bool increase)
         // NOTE: if BHP is below limit, we set state.stop_iteration = true
         auto bhp = state.getBhpWithLimit();
         computeWellRates_(bhp, cur_potentials);
-        auto new_oil_rate = getOilRateWithLimit_(cur_potentials);
-        auto new_gas_rate = getGasRateWithLimit_(cur_potentials);
+        double new_oil_rate = 0.0;
+        bool oilIsLimited = getOilRateWithLimit_(new_oil_rate, cur_potentials);
```

Comment:
I would suggest using snake case style for the variables, since that has been used for all other variables in this file. So instead of `oilIsLimited` use `oil_is_limited`. And similar for `gasIsLimited` below.

Reply:
done